README.md

```diff
@@ -54,4 +55,4 @@ The Security Working Group uses [GitHub issues](https://github.com/webpack/secur
 
 ## Code of Conduct
 
-The [Webpack Project's CoC](https://github.com/webpack/webpack/blob/main/CODE_OF_CONDUCT.md) applies to this repo.
\ No newline at end of file
+The [Webpack Project's CoC](https://github.com/webpack/webpack/blob/main/CODE_OF_CONDUCT.md) applies to this repo.
```

Comment:
I can revert this if needed, but there _should_ be a newline at the end of the file